evil-surround.el

```diff
@@ -105,8 +133,9 @@ Each item is of the form (OPERATOR . OPERATION)."
 (defun evil-surround-function ()
   "Read a functionname from the minibuffer and wrap selection in function call"
   (let ((fname (evil-surround-read-from-minibuffer "" "")))
-    (cons (format "%s(" (or fname ""))
-          ")")))
+    (cons (format (if (member major-mode evil-surround-lisp-modes)
+		      "(%s " "%s(")
```

Comment:
I see what you are doing here. I like it. Adding a test would greatly improve your PR.

Reply:
please review the test, i'm not shure if it's done right.

Reply:
I think the test is fine. not sure about the c-mode activation.